tests/integration/security/ca_custom_root/main_test.go

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package mtlscertplugincasecurenaming
+// cacustomroot creates cluster with custom plugin root CA (samples/cert/ca-cert.pem)
+// instead of using the auto-generated self-signed root CA.
+package cacustomroot
 
```

Comment:
Can we keep the original test for plugin cert only, and add a new test to cover this trust domain validation?

Reply:
why? the old test is not changed, we just use the same cluster.

Reply:
Got it. Thanks.